### PR TITLE
BaSP-HOTFIX/MR-240/Fix activities and add animation

### PR DESCRIPTION
**Pull request by FontanR**
Now actitivies dont show the isActive value either inside the form or the list
Fix validations (server too)
Fix comma issue in the list of trainers from activities
Add animation to activities in members view

### CODE REVIEW

File: src/Components/Admins/Activities/Table/index.jsx [hunk 1]
```diff
@@ -20,6 +20,8 @@ const Table = () => {
     setFilter(activitiesToShow);
   };
 
+  console.log('lsita', list);
```
Comment (WalterArielPavicich) [inline src/Components/Admins/Activities/Table/index.jsx]: the console.log should be removed
```suggestion
```

**Review by jramire5 — APPROVED** [review]
Good Job!

**Review by IgnacioGalceran — APPROVED** [review]
nice!

**Comment by JulianRiedinger7** [comment]
Tests Passed! Great work!

**Pull request merged**